### PR TITLE
Add an appendix on the bootstrap

### PR DESCRIPTION
**Pull request by malcolmbarrett**
This PR moves the callout on the bootstrap from chapter two to a new appendix. It then expands on common questions we get about the bootstrap

### CODE REVIEW

File: appendices/A-bootstrap.qmd [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+    color = "firebrick"
+  ) + 
+  facet_wrap(~ sample)
+```
+
+You may notice that sampling from the population bears a similarity to the sampling we do in the bootstrap.
+We're treating the sample as representative of the population from which it comes.
+This helps us translate the distribution we see into terms of the original population, much the same way we get when we use parametric confidence intervals.
+A key difference from sampling the population is that the bootstrap will determine the spread around the *sample* estimate, not the population estimate.
+Let's look at sample 8 a bit closer.
```
Comment (LucyMcGowan) [inline appendices/A-bootstrap.qmd:241]: Hmm maybe you can ignore my comment above it seems like you use sample throughout for a single iteration of the bootstrap

**Review by LucyMcGowan — APPROVED** [review]
This is so great. I love it!

File: chapters/15-g-comp.qmd
[the remark this answers is not in the document]
Comment (malcolmbarrett) [reply chapters/15-g-comp.qmd:1]: Me too. I feel like I don't need to explain as much about the `rsplit` object now

File: appendices/A-bootstrap.qmd (hunk 1 above)
Comment (malcolmbarrett) [reply appendices/A-bootstrap.qmd:241]: I do use the word "sample", drawing on the notation of emulating sampling from the population. But maybe "resample" would be more precise?

**Pull request merged**